src/Cluster/Cluster.ts

```diff
@@ -1,4 +1,5 @@
-import { Worker, fork } from 'cluster';
+import cluster, { Worker } from 'cluster';
+const fork = cluster.fork
```

Comment:
please import fork from cluster directly in the above import statement, no extra const needed